public/app/plugins/datasource/loki/mocks.ts

```diff
@@ -1,17 +1,23 @@
-import LokiDatasource from './datasource';
+import { LokiDatasource, LOKI_ENDPOINT, LEGACY_LOKI_ENDPOINT } from './datasource';
 import { DataSourceSettings } from '@grafana/data';
 import { LokiOptions } from './types';
 import { createDatasourceSettings } from '../../../features/datasources/mocks';
 
 export function makeMockLokiDatasource(labelsAndValues: { [label: string]: string[] }): LokiDatasource {
+  const legacyLokiLabelsAndValuesEndpointRegex = /^\/api\/prom\/label\/(\w*)\/values/;
+  const lokiLabelsAndValuesEndpointRegex = /^\/loki\/api\/v1\/label\/(\w*)\/values/;
+
+  const legacyLokiLabelsEndpoint = `${LEGACY_LOKI_ENDPOINT}/label`;
+  const lokiLabelsEndpoint = `${LOKI_ENDPOINT}/label`;
+
```

Comment:
Nitpick: this is "just" testcode but it starting to get a bit complex maybe we should start thinking about refactoring makeMockLokiDatasource in future PRs.